src/components/Widget/widget.tsx

```diff
@@ -262,7 +262,7 @@ export const PVWidget = (props: PVWidgetComponent): JSX.Element => {
 
   const components = [];
 
-  if (props.actions) {
+  if (props.actions && props.actions.actions.length > 0) {
```

Comment:
Any particular reason for this? Obviously a button with one action is not going to have *much* use for a MenuWrapper but perhaps the added description can be useful sometimes?

Reply:
Greater than zero, not greater than one.

Reply:
So this prevents empty menu wrappers from being shown?